tools/go-cli/go-whisk-cli/commands/sdk.go

```diff
@@ -72,9 +73,18 @@ var sdkInstallCmd = &cobra.Command{
             err = swiftInstall()
         case "ios":
             err = iOSInstall()
+        case "bashauto":
+            err = WskCmd.GenBashCompletionFile(BASH_AUTOCOMPLETE_FILENAME)
+            if (err != nil) {
+                whisk.Debug(whisk.DbgError, "GenBashCompletionFile('%s`) error: \n", BASH_AUTOCOMPLETE_FILENAME, err)
+                errStr := fmt.Sprintf("Unable to generate '%s': %s", BASH_AUTOCOMPLETE_FILENAME, err)
+                werr := whisk.MakeWskError(errors.New(errStr), whisk.EXITCODE_ERR_GENERAL, whisk.DISPLAY_MSG, whisk.NO_DISPLAY_USAGE)
+                return werr
+            }
+            fmt.Printf("ok. To enable command line completion of wsk commands, source the auto completion script (%s) into your bash environment\n", BASH_AUTOCOMPLETE_FILENAME)
```

Comment:
Perhaps change "ok" to "Successfully installed Bash auto-completion," or remove the "ok" altogether.